src/SelfResponse.php

```diff
@@ -27,10 +27,11 @@ class SelfResponse
     /**
      * SelfResponse constructor.
      * @param $response
+     * @param string $idProperty
      */
-    public function __construct($response)
+    public function __construct($response, $idProperty = "id")
```

Comment:
Nit: using single quotes when we do not need string interpolation will give slight performance improvement.

Reply:
Updated it :+1: